notebooks/import_model.ipynb

```diff
@@ -0,0 +1,110 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from transformers import AutoModelForCausalLM, BitsAndBytesConfig, AutoTokenizer\n",
+    "import torch\n",
+    "\n",
+    "print(torch.cuda.is_available())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# model_path = \"codellama/CodeLlama-7b-Instruct-hf\"\n",
+    "# model_path = \"meta-llama/Llama-2-7b-chat-hf\"\n",
```

Comment:
Should these lines be here? If so, what are they for?

Reply:
removed